src/azure/storage/loader.rs

```diff
@@ -80,7 +84,10 @@ impl Loader {
         let workload_identity_token =
             workload_identity_credential::get_workload_identity_token(&self.config).await?;
         match workload_identity_token {
-            Some(token) => Ok(Some(Credential::BearerToken(token.access_token))),
+            Some(token) => Ok(Some(Credential::BearerToken(
+                token.access_token,
+                token.expires_on.unwrap_or("".to_string()),
```

Comment:
We can set to 600s later to enforce token refresh even when expires on not returned.